src/genie/libs/parser/iosxe/tests/ShowIpSlaSummary/cli/equal/golden_output_1_expected.py

```diff
@@ -0,0 +1,60 @@
+expected_output = {
+    'ids': {
+        '1': {
+            'state': 'active',
+            'type': 'tcp-connect',
+            'destination': '123.23.213.32',
+            'rtt_stats': 'RTT=44',
```

Comment:
let's add integer one like below.
```
'rtt_stats': 'RTT=44',
'rtt_stats_msecs': 44,
```
the reason is, I believe user want to check/compare the value as integer. for that purpose, `rtt_stats_msecs` is useful. but we need to have str one because we don't know what kind of pattern is there.

Reply:
Added `rtt_stats_msecs` as an `Optional()` key in schema. If no numbers exist in the `rtt_stats` key's value, then `rtt_stats_msecs` key is omitted. 

For example:
```
        '101': {
            'state': 'active',
            'type': 'udp-jitter',
            'destination': '192.0.2.2',
            'rtt_stats': '100',
            'rtt_stats_msecs': 100,
            'return_code': 'OK',
            'last_run': '22:49:53 PST Tue May 3 2011'
        },
        '102': {
            'state': 'active',
            'type': 'tcp-connect',
            'destination': '192.0.2.2',
            'rtt_stats': '-',
            'return_code': 'NoConnection',
            'last_run': '22:49:53 PST Tue May 3 2011'
        },
```